src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.cs

```diff
@@ -323,16 +323,6 @@ public static partial class RoslynInsertionTool
                         return false;
                     }
 
-                    try
-                    {
-                        await QueueBuildPolicy(pullRequest, "[Deprecated] ValBuild RPS");
-                    }
-                    catch (Exception ex)
-                    {
-                        Console.WriteLine($"Unable to create a deprecated validation build for '{pullRequest.SourceRefName}'");
-                        Console.WriteLine(ex);
-                    }
-
                     try
                     {
                         await QueueBuildPolicy(pullRequest, "CloudBuild - RPS");
```

Comment:
@dpoeschl @brettfo I don't see an entry for "CloudBuild - DDRITs". Do you know Is that triggered implicitly  somehow or is that being triggered via the deprecated policy that I am removing above?

Reply:
```suggestion
                        await QueueBuildPolicy(pullRequest, "CloudBuild - Request RPS");
```

Reply:
```suggestion
                         await QueueBuildPolicy(pullRequest, "CloudBuild - Request RPS");
```

Reply:
Thanks! Have fixed that.